cli/admin.js

```diff
@@ -1,10 +1,9 @@
 const inquirer = require("inquirer");
-const abiDecoder = require("../../../common/AbiUtils.js").getAbiDecoder();
 const style = require("./textStyle");
-const { isAdminRequest, getAdminRequestId, decodeTransaction } = require("../../../common/AdminUtils.js");
+const { isAdminRequest, getAdminRequestId, decodeTransaction } = require("../common/AdminUtils.js");
+const Governor = require("@umaprotocol/core/build/contracts/Governor.json");
 
 async function decodeGovernorProposal(artifacts, id) {
-  const Governor = artifacts.require("Governor");
   const governor = await Governor.deployed();
```

Comment:
I don't think this works. Because you're requiring a JSON file and calling `deployed()` on it. You need to instantiate a contract instance first, which is what `artifacts.require()` do for you under the hood.

Reply:
Yeah this is a TODO (see PR description). I'll have to replace all `deployed()` and use `web3` to construct the contracts